bibliopixel/drivers/SimPixel/websocket.py

```diff
@@ -81,3 +87,11 @@ def remove_client(self, client):
             self.clients.remove(client)
         except:
             pass
+
+
+PORT_BELOW_1024_ERROR = """
```

Comment:
Move above it's usage. I know python lets you do this. But I look *up* for variables, not down.

Reply:
I'll change it, but there's a reason for this.

Basically, I go from "interesting to boring" - because I am writing for the reader of the code - who will be "me" in a few weeks as I forget this.

I want to start off with the key, must-know, seriously-important stuff at the very top, and then put boring things at the end.

The reason that `PORT_BELOW_1024_ERROR` is at the very bottom is that it's about the most boring thing there.  When you see `PORT_BELOW_1024_ERROR` in the code above, you don't say, "Oh, my gosh, what could that be? I need to know!"  :-D  

So to me, putting it at the top of the page forces the reader to read this boring error message _first_ before the actual important stuff happens.